clang/lib/CodeGen/CGLoopInfo.h

```diff
@@ -360,26 +354,21 @@ class LoopInfoStack {
 
   void addIVDepMetadata(const ValueDecl *Array, llvm::Instruction *GEP);
 
-  /// Set value of an initiation interval for the next loop pushed.
-  void setSYCLIInterval(unsigned C) { StagedAttrs.SYCLIInterval = C; }
+  /// Add value of an initiation interval for the next loop pushed.
+  void addSYCLIInterval(unsigned C) { StagedAttrs.SYCLIInterval = C; }
```

Comment:
This one should not be renamed -- it's still a "set" operation rather than an "add" operation. "Add" should only be for cases where we're collecting more than one value. (Same applied to other changes here.)

Reply:
Done